panedr/panedr.py

```diff
@@ -323,6 +329,20 @@ def add_blocks(self, final_number):
             self.nblock_alloc = final_number
 
 
+class GMX_Unpacker(xdrlib.Unpacker):
+    """xdrlib.Unpacker subclass that implements `unpack_real`
+
+    Decision on whether to return 32- or 64-bit reals is controlled by the
+    `gmx_double` attribute, set to ``False`` by default.
+    """
+    gmx_double = False
```

Comment:
Should it be a class attribute? What happens if you try to read a single and a double precision file one after the other?